virttest/bootstrap.py

```diff
@@ -350,6 +351,15 @@ def _forced_or_detected(forced, detected):
                                       "u%s" % detected.release)
         arch = _forced_or_detected(options.vt_host_distro_arch,
                                    "Host_arch_%s" % detected.arch)
+        vendor = cpu.get_cpu_vendor_name()
+        revision = ""
+        if 'powerpc' in cpu.get_cpu_arch():
+            revision = cpu.get_cpu_version()
```

Comment:
This will fail with older Avocado. How about using:

```
if hasattr(cpu, 'get_cpu_version'):
    revision = cpu.get_cpu_version()
else:
    revision = "unknown"
```